logic/peers.js

```diff
@@ -254,6 +254,24 @@ Peers.prototype.list = function(normalize) {
 	);
 };
 
+/**
+ * Returns a random list of connected peers.
+ *
+ * @param {Object} [options] - Optional
+ * @param {number} [options.limit] - Maximum number of peers to get; defaults to all
+ * @returns {peer[]} List of peers
+ */
+Peers.prototype.listRandomConnected = function(options) {
+	options = options || {};
+	const peerList = Object.keys(self.peersManager.peers)
+		.map(key => self.peersManager.peers[key])
+		.filter(peer => peer.state === Peer.STATE.CONNECTED);
+	const shuffledPeerList = _.shuffle(peerList);
+	return options.limit === undefined
```

Comment:
Why `=== undefined`? Why not just `options.limit` (`undefined`, `null`, `0`, etc.)?

Reply:
@4miners I wanted to account for both `null` and `undefined` but not `0` (that's why I used `x == null` initially).

I can change it to check only for truthiness though (treat 0 as the same as undefined) if that's consistent with the rest of the code.